website/pages/docs/reference/cli/cloudquery.md

```diff
@@ -17,13 +17,13 @@ Find more information at:
 ### Options
 
 ```
-      --color string           Enable colorized output (on, off, auto) (default "auto")
```

Comment:
Since we marked the flag as deprecated it's removed from the help, but still works